.github/workflows/schema-compose.yaml

```
@@ -0,0 +1,93 @@
+name: Schema
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  generate:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4.1.4
+
+      - name: Install Rover CLI
+        run: |
+          curl -sSL https://rover.apollo.dev/nix/v0.23.0-rc.3 | sh
+          echo "$HOME/.rover/bin" >> $GITHUB_PATH
+
+      - name: Compose Schema
+        run: |
+          rover supergraph compose --config schema/supergraph-config.yaml --elv2-license=accept > supergraph.graphql
```

Just makes this a bit more readable
```suggestion
      - name: Compose Schema
        run: >
          rover supergraph compose
          --config schema/supergraph-config.yaml
          --elv2-license=accept
          > supergraph.graphql
```